todo_app/app.py

```diff
@@ -8,4 +12,38 @@
 
 @app.route('/')
 def index():
-    return 'Hello World!'
+    items = get_items()
+
+    items = sorted(items, key=lambda items : items['status'], reverse=True)
+
+    return render_template('index.html', ToDos = items)
+
+@app.route('/submit', methods=['POST','GET'])
+def submit():
+    add_item(request.form.get('title'))
+    return redirect('/')
+
+@app.route('/todo/<id>')
+def view_item(id):
+    item = get_item(id)
+    not_started = False
+    started = False
+    done = False
+
+    if item['status'] == "Not Started":
+        not_started = True
+    if item['status'] == "Started":
+        started = True
+    if item['status'] == "Done":
+        done = True     
```

Comment:
This can be simplified while keeping identical functionality:

```suggestion
    not_started = item['status'] == "Not Started"
    started = item['status'] == "Started"
    done = item['status'] == "Done":
```

Because `item['status'] == "Done"` is an expression that evaluates to `True` or `False`. You aren't limited to using it in an `if` statement.